pkg/db/v1beta1/postgres/postgres_test.go

```diff
@@ -137,3 +137,13 @@ func TestGetDbName(t *testing.T) {
 	}
 
 }
+
+func TestSslMode(t *testing.T) {
+	sslMode := "require"
+	t.Setenv(common.PostgreSslMode, sslMode)
+
+	dbName := "host=katib-postgres port=5432 user=katib password= dbname=katib sslmode=require"
+	if getDbName() != dbName {
+		t.Errorf("getDbName returns wrong value %v", getDbName())
+	}
+}
```

Comment:
I don't think that such a test is useful. If we want to add unit tests, we should expand the above, "TestGetDbName".

Reply:
Done, move to `TestGetDbName` and expand it.